llpc/docs/amdllpc.md

```diff
@@ -4,7 +4,8 @@ LLPC can be built into a standalone offline compiler (amdllpc). It supports GLSL
 
 ## Build instructions
 
-LLPC is normally built as part of the [AMD Open Source Driver for Vulkan](https://github.com/GPUOpen-Drivers/AMDVLK/blob/dev/README.md). The build includes standalone `lgc` and `amdllpc`. You can build `lgc amdllpc` only or build `check-lgc check-lgc-units check-amdllpc check-amdllpc-units` to run local tests besides the build (**Note:** You need to add the option `-DXGL_BUILD_TESTS=ON` in the AMDVLK cmake command before building these local test targets).
+LLPC is normally built as part of the [AMD Open Source Driver for Vulkan](https://github.com/GPUOpen-Drivers/AMDVLK/blob/dev/README.md). The building includes standalone `lgc` and `amdllpc` (**Note:** You need to add the option `-DXGL_BUILD_TOOLS=ON` in the AMDVLK cmake command before building `amdllpc`).
```

Comment:
nit: I'd change 'The building includes' -> 'The build includes'

Also, is the note correct? Since the CMake option is ON by default, shouldn't we explain how to *disable* standalone tools if someone does not want them?

Reply:
xgl will force it to be off by default. So, when building xgl, you need to set XGL_BUILD_TOOLS=ON (which is off by default) and it will force LLPC_BUILD_TOOLS=XGL_BUILD_TOOLS, enabling it again.

IIRC, the whole purpose is to not need spvgen with the default build options, because spvgen needs additional dependencies.

Reply:
Shouldn't we update the amdvlk docker file to set this on then (in this PR)?

Reply:
After the XGL option becomes available, we should then update the dockerfile to set the XGL option

Reply:
docker/amdvlk.Dockerfile updated, could you please review it?

Reply:
LGTM